packages/dialtone-vue2/components/dropdown/dropdown_default.story.vue

```diff
@@ -52,6 +46,13 @@
         {{ item.name }}
       </dt-list-item>
     </template>
+    <template #footer="{ close }">
+      <div class="d-px12">
+        <dt-button @click="close">
+          Dropdown footer
+        </dt-button>
+      </div>
+    </template>
```

Comment:
For this one you can just go

```
<template #footer>
  <div v-if="$attrs.footer" v-html="$attrs.footer" />
</template>
```

This allows the user to pass in whatever html tags they want to it from storybook (empty by default).